api/v1alpha1/istio_types.go

```diff
@@ -200,7 +200,13 @@ const (
 // +kubebuilder:printcolumn:name="Version",type="string",JSONPath=".spec.version",description="The version of the control plane installation."
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="The age of the object"
 
-// Istio represents an Istio Service Mesh deployment
+// Istio represents an Istio Service Mesh deployment consisting of one or more
+// control plane instances (represented by one or more IstioRevision objects).
+// To deploy an Istio Service Mesh, a cluster-admin creates an Istio object with
+// the desired Istio version and configuration. The Istio operator then creates
+// an IstioRevision object, which in turn creates the underlying Deployment
+// objects for istiod and other control plane components, not unlike how a
```

Comment:
"not unlike" sounds confusing to me. Perhaps "similar to?"

Reply:
Yes, confusing to us non-native speakers, but I think native speakers usually use this form instead (even though it is a double negative). Let's wait for @rh-tokeefe to chime in.

Reply:
@jwendell and @luksa in a way, both of you are correct. The phrase "not unlike" is correct, but it isn't all that common. I think  Jonh's suggestion works well. "...control plane components, similar to how a Deployment object in Kubernetes creates ReplicaSets that create the Pods"

Reply:
Oh, interesting. I thought it was common, but maybe that's because I always notice it because of the double negative. I'll change it to "similar to".